client/js/actions/snippet-actions.js

```diff
@@ -8,14 +8,15 @@ class SnippetActions {
     //   'create'
     // );
   }
+
   getAll () {
-    axios.get('/snippets')
+    return axios.get('/snippets')
```

Comment:
@vrael560 first of all on server response you will change a ```Store```. And in your component you can  ```listen``` to ```Store``` changes.
```javascript
Store.listen((data) => {/*...*/})
```